charts/pega/values.yaml

```diff
@@ -137,6 +137,7 @@ global:
 
       hpa:
         enabled: true
+        memoryTarget: false
```

Comment:
Should be `enableMemoryTarget` right?

Reply:
You are right. I just pushed a commit which fixes that.